packages/vite-plugin-cloudflare/e2e/fixtures/basic/package.json

```diff
@@ -14,7 +14,7 @@
 		"react-dom": "^19.0.0"
 	},
 	"devDependencies": {
-		"@cloudflare/vite-plugin": "https://prerelease-registry.devprod.cloudflare.dev/workers-sdk/runs/13293829928/npm-package-cloudflare-vite-plugin-8118",
+		"@cloudflare/vite-plugin": "0.x",
```

Comment:
Should this be the workspace package version? I'm not sure exactly how this works with the custom registry. Does this correctly test against code changes in a PR?

Reply:
This fixture is copied to a temporary directory outside the workspace so we cannot use `workspace:*` type dependencies.
What will happen is that we will run `npm install` (or equivalent) in the temporary directory but the verdaccio npm registry will intercept the requests and try to handle them itself before falling back to the public registry.
The idea he was to try to get the most recent vite-plugin package, which should be the one we published to the verdaccio registry.

Reply:
In that case, shouldn't the version be `*`? Want to make sure it's not something we might forget to update when we move to 1.0.

Reply:
I did try * previously and it failed. But just tried again now and it seems better.